static/docs/install/completion.md

```diff
@@ -2,6 +2,12 @@
 
 Enjoy working with DVC faster and with fewer typos!
 
+> Note that shell completion is automatically enabled when DVC is installed on
```

Comment:
We can make it just a regular paragraph, what do you think?

Reply:
You're right, updated!